samza-core/src/test/java/org/apache/samza/operators/TestMessageStreamImpl.java

```diff
@@ -18,6 +18,15 @@
  */
 package org.apache.samza.operators;
 
+import java.util.ArrayList;
```

Comment:
Yeah, but a lot of existing files have the previous order. Not a big deal, it's already inconsistent across files.